docs/graasp-insights-user-guide.md

```diff
@@ -0,0 +1,243 @@
+
+# Results
+
+The `Results` tab displays the datasets resulting from executing algorithms.
+
+![Results tab](./figures/results.png)
+
+Similar to the `Datasets tab`, various information about a result is displayed and its content can be inspected with the `View result` icon.
+
+After having certified that the dataset was properly anonymized using the [Validations](#Validations) screen, you can export it into your file system by clicking on the `Export result` icon (floppy disk).
+
+# Schemas
+
+Navigate to the `Schemas` tab to see every schema that is defined in Graasp Insights.
+
+![Schemas tab](./figures/schemas.png)
+
+A schema describes the content structure of a dataset. The datasets loaded in Graasp Insights will have a schema tag attached to it if it satisfies the schema definition.
+
+Generic algorithms will usually have a field selection that is fully dependent on the definition of schemas. Therefore, if a dataset doesn't follow any schema, it is advised to (automatically) generate one through the `Schemas` tab.
+
+Click on the lower right `+` button to generate a new schema.
+
+![Add Schema](./figures/add-schema.png)
+
+Enter a label, a description, a dataset to generate the schema from and a color for the schema tag. When done, click on `ADD SCHEMA`. If no dataset is selected, then the generated schema will be empty.
+
+![Schemas tab with new schema](./figures/schemas-with-new-schema.png)
+
+Click on the `View Schema` button to inspect the generated schema.
+
+![View Schema](./figures/view-schema.png)
+
+The schema definition on the left, the label, the description and the tag color can all be modified within this page. A sensible attribute to investigate is the `"required"` attribute which dictates which fields are required for the dataset to be considered valid. Click [here](https://json-schema.org/learn/getting-started-step-by-step.html) to learn more about `JSON` schemas.
+
+Do not forget to `SAVE` your modifications before leaving the page.
+
+# Adding an algorithm
+
+From the `Algorithms` tab, add a new algorithm with the lower right `+` button.
+
+A new screen titled `Add Algorithm` shows up with three alternatives to add a new algorithm:
+
+1. > Load algorithm from a file
+
+Add an algorithm from your file system. Although any `python` algorithm can be added, dataset loading and saving as well as parameter handling is done in a specific way. It is therefore recommended having a look at existing algorithms to have a better understanding of how the algorithm should communicate with the interface. The `Write algorithm` option is also a good alternative to quickly learn about what the algorithm layout should look like.
+
+![Load algorithm from a file](./figures/load-algorithm-from-file.png)
+
+2. > Add Graasp algorithm
+
+Select a premade Graasp algorithm to add to the application. Those are already present when first launching Graasp Insights. If you delete a pre-existing algorithm, you can add it back with this option.
```

Comment:
_pre-made_

Reply:
also, just to make sure: When the application first loads, does it include the Graasp algorithms/schema, or do they have to be loaded in via the Settings tab?